MapboxNavigation/Style.swift

```diff
@@ -32,10 +32,24 @@ open class Style: NSObject {
     @objc public var styleType: StyleType = .day
     
     /**
-     Map style to be used for the style.
+     URL of the style to display on the map during turn-by-turn navigation.
      */
     @objc open var mapStyleURL: URL = MGLStyle.navigationGuidanceDayStyleURL
     
+    #if canImport(CarPlay)
```

Comment:
Do we need to wrap this in a canImport? The URL is still available iOS < 12.

Reply:
The `#else` block below is identical except for the documentation comment. I figured it would make little sense to talk about CarPlay in the documentation if CarPlay is unavailable.